org.freedesktop.fldigi.yml

```diff
@@ -0,0 +1,52 @@
+app-id: org.freedesktop.fldigi
```

Comment:
The same question as your old PR, I don't think you can use org.freedesktop as a prefix for your app-id because it's not part of freedesktop ?

Reply:
Tell me how it should be. This is my first flatpak and I don't understand English. I try to translate all the manuals I can, but there are things that escape when translating it. Help me with this to correct it.
I have many programs that are used for hobby radio that I want
include
I do this because many are older people and are starting with linux, when using flatpak the programs run practically in any distro
I would appreciate your help, thank you

Reply:
It seems like the program's main website is at http://www.w1hkj.com/, so com.w1hkj.fldigi would be appropriate (not sure if digits would be valid in app ids though).

Reply:
ok tks -- 
should I rename the files too?
![Captura de pantalla_2020-11-29_18-44-14](https://user-images.githubusercontent.com/30733717/100554355-edab0200-3272-11eb-80d4-fac8a72a49e9.png)

Reply:
Yes, make sure to update the file names and build commands in the yml file too.

Reply:
ok perfect tks